app/models/ext_management_system.rb

```diff
@@ -154,6 +154,7 @@ def self.create_from_params(params, endpoints, authentications)
 
   supports_not :catalog
   supports_not :add_host_initiator
+  supports_not :add_host_initiator_group
```

Comment:
Hey @gregoryb-at-ibm we're trying to get away from new supports features on the EMS that simply track child model standard create/update/delete features.  Correct me if I'm wrong but `ems.supports?(:add_host_initiator_group)` could be `ManageIQ::Providers::Autosde::StorageManager::HostInitiatorGroup.supports?(:create)` ?  If so we have a pattern for that where the EMS delegates the supports check to the child model and then we don't have to introduce more unique features